### ISSUE TITLE
What is the LoRaWAN version? 1.0.x?

### ISSUE THREAD

**Issue by tibbis** [matthijskooijman/arduino-lmic#307]
What is the LoRaWAN version of this library? 1.0.x?
thanks

**Comment by matthijskooijman** [comment]
Yes, this is mostly the unchanged original LMIC library, which predates 1.x, so it is definitely 1.0.x. I'm not entirely sure which revision exactly, though.

**Comment by tibbis** [comment]
Ok I need to know what version to set in the TTN portal, e.g. 1.0.0, 1.0.1 or 1.0.2 etc?

**Comment by matthijskooijman** [comment]
TTN seems to select 1.0.2 by default, which works fine for my nodes running this library.
